ui/packages/catalog-ui-search/src/main/webapp/component/singletons/metacard-definitions.js

```diff
@@ -218,7 +224,7 @@ module.exports = new (Backbone.Model.extend({
   },
```

Comment:
does `getMetacardStartingTypes` need to return `metacardStartingTypesWithTemporal()`? is there anywhere else that uses `metacardStartingTypes` that should now use `metacardStartingTypesWithTemporal()` ?

Reply:
I don't believe so